lib/axe/api/a11y_check.rb

```diff
@@ -37,11 +37,18 @@ def call(page)
       private
 
       def audit(page)
-        yield page.execute_async_script "#{METHOD_NAME}.apply(#{Core::JS_NAME}, arguments)", @context.to_json, @options.to_json
+        yield page.execute_async_script "#{METHOD_NAME}.apply(#{Core::JS_NAME}, arguments)", *js_args
+      end
+
+      def js_args
+        [ @context, @options ]
+          .reject{ |arg| arg.empty? }
```

Comment:
That's surprising, since you can pass an empty object for options to axe-core. Context should default to the document if it doesn't match a selector...but that could be where it's erroring out.